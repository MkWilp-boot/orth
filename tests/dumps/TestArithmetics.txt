100
10
20
50
