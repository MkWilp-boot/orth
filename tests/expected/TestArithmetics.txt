100
100
30
1